src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -552,8 +552,14 @@ protected function getQueryForList() {
   protected function getQueryCount() {
     $query = $this->getEntityFieldQuery();
 
-    // If we are trying to filter on a computed field, just ignore it and log an
-    // exception.
+    // If we are trying to filter or sort on a computed field, just ignore it
+    // and log an exception.
+    try {
+      $this->queryForListSort($query);
```

Comment:
We seem to be doing this twice. Any reason?

Reply:
What do you mean twice? The other queryForListSort() call happens in getQueryForList(), and I added it inside getQueryCount(), which contains almost the same logic, besides the call to $this->queryForListSort($query);

Reply:
Ah! You're right. Never review code before 5:30am.